scripts/etc/init.d/rcS

```diff
@@ -834,6 +835,12 @@ touch /rootfs/etc/network/interfaces || fail
     fi
 } >> /rootfs/etc/network/interfaces || fail
 
+if [ "${disable_predictable_nin}" = "1" ]; then
+    # as described here: https://www.freedesktop.org/wiki/Software/systemd/PredictableNetworkInterfaceNames
+    # adding net.ifnames=0 to /boot/cmdline
+    cmdline="${cmdline} net.ifnames=0"
```

Comment:
> whether this should be an opt-in or an opt-out feature

we could, I guess... I (obviously) think it should be opt-out feature, because of the specifics of the device and likely use cases...

were this a desktop computer, where you can switch network interfaces and who know what, sure, I'd go with the opt-in, but being what it is, however...